readme.md

```diff
@@ -80,7 +80,35 @@ cmake --build . --target install --config Release --parallel 12       #< Build a
 > You can use the config files located at `<SUPERBUILD_INSTALL_DIR>/CT_ICP/lib/cmake` to load the libraries in a cmake
 > project, or use ROS or the specified executables.
 
-### Step 3: ROS
+### Step 3: Python Bindings
+
+> **CT-ICP** comes with python bindings. To build the python bindings, follow the following instructions:
+
+- 1. Install the superbuild (**with pybind11** /!\ )
+- 2. Install `CT_ICP` (see Step 1., with the CMake option `WITH_PYTHON_BINDING` activated) 
```

Comment:
You can install it with the 2nd step instead of the 1st. For me on Ubuntu 20.04 superbuild fails after a successful build of a Eigen3_ext target. But the 2nd option worked alright, and generated a shared library.

Reply:
Okay, maybe it actually doesn't work that easily, huh. I get undefined references for the shared slam core lib...